examples/production/values.yaml

```diff
@@ -23,6 +23,7 @@ spire-server:
       drop: [ALL]
     seccompProfile:
       type: RuntimeDefault
+  priorityClassName: system-node-critical
```

Comment:
Why would the server be critical to a node? A common pattern is to do this for DaemonSets to make room for them on a node, I don't understand why we would want to do this for this statefulset which in essence does not require to run on a specific node.

What is the usecase for this? What are your trying to achieve?

Reply:
Maybe the example that I chose was not optimal. Instead, I should have chosen `system-cluster-critical`.

The idea is that `spire-server` is critical to our cluster and should not be evicted before other, less critical, pods. If we were unable to set the `priorityClassName`, it could happen that the `spire-server` gets evicted and it would no longer be possible to start new pods since they need the `spire-server` to get their SVIDs.

Reply:
💡 that makes more sense, thanks for clarifying the usecase.

Could you update the PR with that change?

Reply:
Sure thing, it should be updated now :+1: